### PR TITLE
add alt_label

### PR DESCRIPTION
**Pull request by stepps00**
Fixes a portion of https://github.com/whosonfirst-data/whosonfirst-data/issues/1714.

More complicated source examples:

- data/890/458/697/890458697-alt-can-databc.geojson
- data/856/330/41/85633041-alt-quattroshapes.geojson
- data/858/952/65/85895265-alt-quattroshapes_pg.geojson
- data/890/456/813/890456813-alt-can-databc.geojson

### CODE REVIEW

**Comment by stepps00** [comment]
The script to add `src:alt_label` properties to each repo was tested on this repo, which didn't pass two alt files the correct properties - this was fixed in f5f35ab.

This PR can be merged once approved.

**Review by nvkelso — APPROVED** [review]
LGTM

**Pull request merged**
